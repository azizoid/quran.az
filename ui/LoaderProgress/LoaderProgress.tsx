import styles from './LoaderProgress.module.scss'

export const LoaderProgress = () => <div className={styles.progressLoader}></div>