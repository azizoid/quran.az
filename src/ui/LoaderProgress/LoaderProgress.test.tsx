import { render } from '@testing-library/react'

import { LoaderProgress } from './LoaderProgress'

test('render the snapshot of LoaderProgress', () => {
  const { container } = render(<LoaderProgress />)

  expect(container).toMatchSnapshot()
})
