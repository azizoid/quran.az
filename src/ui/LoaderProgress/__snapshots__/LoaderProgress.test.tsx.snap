// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the snapshot of LoaderProgress 1`] = `
<div>
  <div
    class="progressLoader"
  />
</div>
`;
